src/TestInfrastructure/Framework/ConfigurableSolutionRuleSetsInformationProvider.cs

```diff
@@ -31,11 +30,24 @@ IEnumerable<RuleSetDeclaration> ISolutionRuleSetsInformationProvider.GetProjectR
 
             return result;
         }
+        string ISolutionRuleSetsInformationProvider.GetSolutionSonarQubeRulesFolder()
+        {
+            return Path.Combine(this.SolutionRootFolder, Constants.SonarQubeManagedFolderName);
+        }
 
-        string ISolutionRuleSetsInformationProvider.CalculateSolutionSonarQubeRuleSetFilePath(string sonarQubeProjectKey, string fileNameSuffix)
+        string ISolutionRuleSetsInformationProvider.CalculateSolutionSonarQubeRuleSetFilePath(string sonarQubeProjectKey, RuleSetGroup group)
         {
-            return Path.Combine(this.SolutionRootFolder, Constants.SonarQubeManagedFolderName, sonarQubeProjectKey + fileNameSuffix + "." + Constants.RuleSetFileExtension);
+            string fileName = sonarQubeProjectKey + group.ToString() + "." + Constants.RuleSetFileExtension;
```

Comment:
Would `$"{sonarQubeProjectKey}{group}.{Constants.RuleSetFileExtension}";` not be clearer to see the structure?